### PR TITLE
Update common functions

### PR DESCRIPTION
**Pull request by dtrunov**
Create method for retrieving data from the database.
Rework method is_db_contains

### CODE REVIEW

**Comment by dtrunov** [comment]
@VVeremjova please review

File: user_modules/shared_testcases/commonFunctions.lua [hunk 1]
```diff
@@ -881,46 +881,49 @@ function os.capture(cmd, raw)
...
-  else
-    return ret_value
-  end
-end
-
----------------------------------------------------------------------------------------------
---18. Function checks value of column from DB with input data
----------------------------------------------------------------------------------------------
-function commonFunctions:is_db_contains(db_path, sql_query, exp_result)
+function commonFunctions:get_data_policy_sql(db_path, sql_query)
```
Comment (VVeremjova) [inline user_modules/shared_testcases/commonFunctions.lua]: @dtrunov Please add doxygen comment

File: user_modules/shared_testcases/commonFunctions.lua [hunk 2]
```diff
@@ -881,46 +881,49 @@ function os.capture(cmd, raw)
...
 	if string.match(sql_query, "^%a+%s*%*%s*%a+") ~= nil then
```
Comment (VVeremjova) [inline user_modules/shared_testcases/commonFunctions.lua]: @dtrunov Please check indents

**Comment by dtrunov** [comment]
@VVeremjova Please review latest changes

**Comment by dtrunov** [comment]
@LuxoftAKutsan Please merge

**Pull request merged**